scipy/optimize/tests/test_optimize.py

```diff
@@ -1891,9 +1891,12 @@ def test_attributes_present(self):
                 if method in skip and attribute in skip[method]:
                     continue
 
-                assert_(hasattr(res, attribute))
+                assert hasattr(res, attribute)
```

Comment:
Just curious - why the change?

Reply:
I think the "informal decision" the last time this came up was that it was "ok," perhaps encouraged, to change from the old-style function to the plain `assert` now that we use `pytest` (which has shims to prevent optimizing out of asserts) in cases where you happen to see it or add a new usage, but that we probably don't want to do a mass auto-reformat.